Jenkinsfile

```diff
@@ -271,10 +255,28 @@ pipeline {
     }
 }
 
+def makeConfig(String filename) {
+    //def spalloc = 'spalloc://Jenkins@10.11.192.11:22246'
+    def spalloc = 'spalloc://Jenkins@spinnaker.cs.man.ac.uk:22246'
```

Comment:
Note - the internal IP address is used as both the Jenkins server and the machine have addresses on this internal network.  Using the external IP means that traffic is likely to flow out of and in to the machine room i.e. the tests become more dependent on the networking state outside of the machine room.

Reply:
Yes, but tests fail catastrophically if I do that. I'm guessing there's some sort of iptables issue causing trouble. (Configuring `/etc/hosts` is better than using hardcoded IP addresses.)